src/Compilers/VisualBasic/Test/Semantic/Binding/Binder_Expressions_Tests.vb

```diff
@@ -3041,7 +3037,7 @@ Class Program
             Dim x As Object = ""
             Color.M(x)
         Catch e As Exception
-            Console.WriteLine(e)
+            Console.WriteLine(e.GetType())
```

Comment:
I agree that a call stack is not that interesting. However, I believe there is a benefit in checking the massage to ensure that the exception is thrown by the Overload Resolution in VB run-time rather than thrown due to some other reason. Consider setting CurrentCulture and CurrentUICulture to invariant before this Try and restoring them after. Then we can safely assert the message as well.